doc_build_aliases.sh

```diff
@@ -207,6 +207,8 @@ alias docbldecsjv='$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-logging-java/do
 
 alias docbldecsnet='$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-dotnet/docs/index.asciidoc --resource=$GIT_HOME/ecs-logging/docs/ --chunk 1'
 
+alias docbldecsjs='$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-logging-js/docs/index.asciidoc --resource=$GIT_HOME/ecs-logging/docs/ --chunk 1'
```

Comment:
```suggestion
alias docbldecsjs='$GIT_HOME/docs/build_docs --doc $GIT_HOME/ecs-logging-nodejs/docs/index.asciidoc --resource=$GIT_HOME/ecs-logging/docs/ --chunk 1'
```

Reply:
Perhaps s/docbldecsjs/docbldecsnodejs/ as well, but I don't know if that matters.

Reply:
Doesn't hurt.